commands/operator-sdk/cmd/scorecard.go

```diff
@@ -20,11 +20,31 @@ import (
 
 	log "github.com/sirupsen/logrus"
 	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
 
 	"github.com/operator-framework/operator-sdk/commands/operator-sdk/cmd/scorecard"
 	"github.com/operator-framework/operator-sdk/version"
 )
 
+// config stores all scorecard config passed as flags
+type config struct {
+	namespace          string
+	kubeconfigPath     string
+	initTimeout        int
+	csvPath            string
+	basicTests         bool
+	olmTests           bool
+	tenantTests        bool
+	namespacedManifest string
+	globalManifest     string
+	crManifest         string
+	proxyImage         string
+	proxyPullPolicy    string
+	verbose            bool
+}
+
+var scConf config
```

Comment:
`scConf` isn't used anywhere else. I'd rather break config fields into `const`'s than have global state.

Reply:
The reason we use scConf is because many other commands have similar or identical fields (like kubeconfig and namespace). By making a separate struct and using a name specific to the command, we can avoid var naming conflicts

Reply:
That being said, I'll rename `config` as that is not very descriptive.